code_route/cli/app.py

```
@@ -0,0 +1,1303 @@
+            return
+
+        # Find matching session
+        sessions = await self.session_manager.list_sessions(limit=100)
+        matching = [s for s in sessions if s.id.startswith(session_id)]
+
+        if not matching:
+            self.console.print(f"[red]Session not found: {session_id}[/red]")
+            return
+
+        if len(matching) > 1:
+            self.console.print(f"[yellow]Multiple matches, be more specific[/yellow]")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)